debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnector.java

```diff
@@ -93,7 +93,7 @@ protected void validateConnection(Map<String, ConfigValue> configValues, Configu
                 // Prepare connection without initial statement execution
                 connection.connection(false);
                 testConnection(connection);
-                checkWalLevel(connection, postgresConfig);
+                // checkWalLevel(connection, postgresConfig);
```

Comment:
Let's follow similar conventions as our YSQL code.

In this case, please add a `// YB Note: ` comment just above explaining why we commented this code. Will turn out to be helpful later.

Reply:
Will be introducing a YBPGConnector.IsYugaByteEnabled() and follow a model similar to YSQL

This check validates that the WAL level is "logical". Skipping this as it does not apply to our case.